lib/v2/layoffs/index.js

```diff
@@ -57,12 +44,59 @@ module.exports = async (ctx) => {
         'x-time-zone': 'America/Los_Angeles',
     };
 
+    // Get the latest data source url
+    let dataSourceUrl = await ctx.cache.get(ENTRY_URL);
+    let cacheInvalid = false;
+    let response;
+    if (!dataSourceUrl) {
+        cacheInvalid = true;
+    } else {
+        // Try to fetch data,
+        // it may fail due to outdated url
+        try {
+            response = await got({
+                method: 'get',
+                url: dataSourceUrl,
+                headers,
+            });
+            if (!response.ok) {
```

Comment:
`.ok` is added in got v12, RSSHub uses got v11. You can check the `statusCode` instead.
```js
if (response.statusCode >= 400)
```

Reply:
Fixed, thanks!